test/middleware/company.authentication.middleware.spec.ts

```diff
@@ -192,7 +215,7 @@ describe('Company Authentication middleware', () => {
     expect(logger.errorRequest).toHaveBeenCalledTimes(1);
   });
 
-  test("should log error if no companuy number in trsansaction", async () => {
+  test("should log error if no companuy number in transaction", async () => {
```

Comment:
Would be good to correct spelling of 'company' here too.

Reply:
Updated this.